src/Debug/physicsViewer.ts

```diff
@@ -256,8 +256,19 @@ export class PhysicsViewer {
                     // Handle compound impostors
                     var childMeshes = targetMesh.getChildMeshes().filter((c) => { return c.physicsImpostor ? 1 : 0; });
                     childMeshes.forEach((m) => {
-                        var a = this._getDebugBoxMesh(utilityLayerScene);
-                        a.parent = m;
+                        mesh = this._getDebugBoxMesh(utilityLayerScene);
+                        if (m.physicsImpostor &&  m.physicsImpostor.type === PhysicsImpostor.BoxImpostor && m.getClassName() === "Mesh") {
+                            const boundingInfo = m.getBoundingInfo();
+                            const min = boundingInfo.boundingBox.minimum;
+                            const max = boundingInfo.boundingBox.maximum;
+                            mesh.position = min.clone();

```

Comment:
```suggestion
                            mesh.position.copyFrom(min);

```

Will avoid recreating a vector3 each time